src/main/java/frc/robot/RobotContainer.java

```diff
@@ -219,19 +223,24 @@ private void configureButtonBindings() {
         new Button(m_operatorController::getYButton).whenPressed(() -> climber_.setState(ClimberStates.PREPARE_HIGH));
 
         
+
+        
     // new Trigger(() -> (Math.abs(m_operatorController.getRightTriggerAxis()) > 0.1))
     // .whenActive(climber_.RetractClimber(m_operatorController.getRightTriggerAxis()));
     
 
-        // Run intake: Operator right trigger
+ // Run intake: Operator right trigger
         new Trigger(() -> (Math.abs(m_operatorController.getRightTriggerAxis()) > 0.7))
-                    .whenActive(m_intakeSubsystem::intake)
-                    .whenInactive(m_intakeSubsystem::stopIntake);
+            .whenActive(m_intakeSubsystem::intake)
+            .whenInactive(m_intakeSubsystem::stopIntake);
 
         // Reverse intake: Operator left trigger
         new Trigger(() -> (Math.abs(m_operatorController.getLeftTriggerAxis()) > 0.7))
-                    .whenActive(m_intakeSubsystem::reverseIntake)
-                    .whenInactive(m_intakeSubsystem::stopIntake);
+            .whenActive(m_intakeSubsystem::reverseIntake)
+            .whenInactive(m_intakeSubsystem::stopIntake);
+
+        new Trigger(() -> m_operatorController.getPOV() == 270).whenActive(Intake.getInstance()::intakeZero);
```

Comment:
Follow the standard of having the intake system be initialized at the start as a member variable